packages/smart-wallet/src/smartWallet.js

```diff
@@ -315,10 +351,27 @@ export const prepareSmartWallet = (baggage, shared) => {
         .returns(M.promise()),
       publishCurrentState: M.call().returns(),
       watchPurse: M.call(M.eref(PurseShape)).returns(M.promise()),
+      repairUnwatchedSeats: M.call().returns(),
+      updateStatus: M.call(M.any(), M.string()).returns(),
```

Comment:
this would take UpdateStatusShape if there was one. since this is a helper method I think this is okay